kernel/read.c

```diff
@@ -362,7 +446,7 @@ BOOLEAN pmemFastIoRead (
 	
 	IoStatus->Status = status;
 	IoStatus->Information = 0;
-	return FALSE;
+	return TRUE;

```

Comment:
In short: Winpmem is a very special case for FAST I/O implementation, actually. Maybe it would suffice to say that for winpmem it can always be TRUE. 

TRUE means no `second try` with a traditional IRP wanted.
FALSE means, "please send the **very same** I/O request **again**, but in a _traditional IRP_".
However: a traditional IRP does not help with whatever problem we encountered. Lots of problems may occur, but not because the request data was not served in an IRP. 

Whatever error occurred, we do not want to experience it again. We handle it here. Therefore we return TRUE. That means: we handled it, whether it was a success or error. (The I/O manager can see the status code in the IOSB).  And we do not want to experience it again, because we would not do better on a second try.